site/documentation.html

```diff
@@ -158,6 +158,7 @@ <h2><span class="text-capitalize">Apache Spark<span class="tm">&trade;</span></s
 <p>Setup instructions, programming guides, and other documentation are available for each stable version of Spark below:</p>
 
 <ul>
+  <li><a href="/docs/3.5.4/">Spark 3.5.4</a></li>
```

Comment:
This seems not working so far. Could you double-check, @LuciferYang ?
- https://spark.apache.org/docs/3.5.4/

Reply:
Let me check it

Reply:
Yes, it seems to be ASF Infra side publishing issue.